frontend/src/metabase/databases/components/DatabaseForm/DatabaseForm.unit.spec.tsx

```diff
@@ -0,0 +1,137 @@
+import { render, screen, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+
+import type { Engine } from "metabase-types/api";
+
+import DatabaseForm from "./DatabaseForm";
+
+interface SetupOpts {
+  onSubmit?: () => void;
+}
+
+function setup({ onSubmit }: SetupOpts) {
+  render(<DatabaseForm engines={engines} isAdvanced onSubmit={onSubmit} />);
```

Comment:
The `DatabaseEditApp` set `isAdvanced` to true

https://github.com/metabase/metabase/blob/e3bd73881dd12d845d580eceb9db6bbe40bdd770/frontend/src/metabase/admin/databases/containers/DatabaseEditApp.tsx#L166-L171

Without it, it will render another UI which is used only when we're setting up a new Metabase instance.